buildpack.md

```diff
@@ -902,9 +947,10 @@ name = "<dependency name>"
 ### Layer Content Metadata (TOML)
 
 ```toml
-launch = false
-build = false
-cache = false
+[types]
```

Comment:
Was surprised to see this. I don't think we need to use `types.launch` in every mention, but the `Layer Types` section should make it clear that the keys are nested under this table. That, or we just don't introduce this table in this PR.

Reply:
Hi @jkutner, thank you for your feedback!

I've just added a commit that specifying that these flags are under `[types]` every time that we refer to `<layers>/<layer>.toml`. Please see https://github.com/buildpacks/spec/pull/209/commits/81cb981a51b6186238559a28a327366160fa2024.

At the beginning of the `Layer Types` section, we have a link to `Layer Content Metadata`, where we specify that all of the flags are under `[types]`. Do you think we should make it clearer than that?
I'm afraid that we want to specify it now only because it's new, but actually it's not necessary.